src/modules/Builder/components/ToggleItemContainer/ToggleItemContainer.styles.ts

```diff
@@ -28,15 +28,18 @@ export const StylesTitleWrapper = styled(StyledFlexTopCenter, shouldForwardProp)
     open,
```

Comment:
It appears that the commits from our previous work, which have already been merged into the development branch, are now showing in your pull request. Could you please double-check?

Reply:
How curious, yes, I'm seeing that above:

<img width="916" alt="Screenshot 2024-02-15 at 10 21 00 AM" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1282772/b2a33ed0-318a-4540-9651-727a24423abb">

I have no idea how this happened; all I've been doing is rebasing on `develop`. I'll fix it, thanks for the callout.